docker-compose.yml

```diff
@@ -106,7 +106,7 @@ services:
 
   altinn_repositories:
     container_name: altinn-repositories
-    image: gitea/gitea:1.7
+    #image: gitea/gitea:1.7
```

Comment:
Why is this commented?

Reply:
Should not be. It was only for davids temporary build to fix the "docker layer exceeded" error message.

Reply:
Removed the '#'